OneCommunityGlobal/HighestGoodNetworkApp
# Raul add owner message section to the top

# Description
(DASHBOARD COMPONENT - PRIORITY LOW) 

> (PRIORITY LOW) Jae: Add to top of app a section that the “Owner” class (only) can edit to post a message to all users of the app like, “Happy Holidays”. (WIP - Raul)
> When not showing a custom message, it should show our horizontal logo. Give me the dimensions I’m allowed and I’ll have our graphics guy create one just for this.
> I’d like the UI for posting the message to have the ability for me to set an expiration date so it’ll automatically return to the logo on that date

## Mainly changes explained:
An owner message section was added to the Header. So now, owner accounts can post messages by image or text on the header for all the volunteers.

## How to test:
1. this PR is related to backend [PR #288](https://github.com/OneCommunityGlobal/HGNRest/pull/288). You have to use both to test this feature.
2. git checkout raul_add_owner_section_to_the_top
3. npm install
4. npm run start:local
5. log in as **OWNER**
6. A pen should be displayed for you:
![header](https://user-images.githubusercontent.com/29555732/223524828-d0572c7c-eec8-4a70-a1c9-45a148b955d8.png)

7. click and try to create and delete messages. For image messages, try using this one - which the resolution is made for this resource:
![image](https://user-images.githubusercontent.com/29555732/223524950-192aa6ab-212a-4504-859e-11a4efa6e2a2.png)

8. The pen should only be displayed for owner accounts. The message has to be displayed for everyone.
9. This resource is made only for computers (width higher than 1400px).
